src/settings/InventoryRecordTypeSelectField.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { Field } from 'redux-form';
+import { FormattedMessage } from 'react-intl';
+import PropTypes from 'prop-types';
+
+import { Select } from '@folio/stripes/components';
+
+import { INVENTORY_RECORDS_TYPE } from '../components/POLine/const';
+
+const InventoryRecordTypeSelectField = ({ label, name, defaultValue }) => (
+  <Field
+    component={Select}
+    format={value => value || defaultValue}
```

Comment:
this is kind of confusing. We do have `initialValues`, maybe it makes sense to put default value there and this hackish `format` could be omitted.

Reply:
+1

Reply:
Done